website/templates/home.html

```diff
@@ -0,0 +1,66 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+{% if user.is_authenticated %}
+
+    <table class="table table-striped table-hover table-bordered">
+        <thead class="table-dark">
+        <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Email</th>
+            <th scope="col">Phone</th>
+            <th scope="col">Address</th>
+            <th scope="col">City</th>
+            <th scope="col">State</th>
+            <th scope="col">Zipcode</th>
+            <th scope="col">Created At</th>
+            <th scope="col">ID</th>
+        </tr>
+        </thead>
+        <tbody>
+        
+            {% if records %}
+                {% for record in records %}
```

Comment:
Default value for records should be empty list this way you will avoid if condition